pkg/rootless/rootless_linux.go

```diff
@@ -325,7 +325,7 @@ func becomeRootInUserNS(pausePid, fileToRead string, fileOutput *os.File) (_ boo
 		uidsMapped = err == nil
 	}
 	if !uidsMapped {
-		logrus.Warnf("Using rootless single mapping into the namespace. This might break some images. Check /etc/subuid and /etc/subgid for adding sub*ids")
+		logrus.Warnf("Using rootless single mapping into the namespace. This might break some images. Check /etc/subuid and /etc/subgid for adding sub*ids if configured locally")
```

Comment:
Maybe "if not using a network user"? I worry that "configured locally" is not very clear

Reply:
sure, that sounds better.  Amended and pushed a new version